node/engine/chainservice/simulated_backend_chainservice_test.go

```diff
@@ -123,16 +145,16 @@ func TestSimulatedBackendChainService(t *testing.T) {
 	}
 	// Check that the recieved event matches the expected event
 	concludedEvent := <-out
-	expectedEvent := ConcludedEvent{commonEvent: commonEvent{channelID: cId, blockNum: 5}}
-	if diff := cmp.Diff(expectedEvent, concludedEvent, cmp.AllowUnexported(ConcludedEvent{}, commonEvent{})); diff != "" {
+	expectedEvent2 := ConcludedEvent{commonEvent: commonEvent{channelID: cId, blockNum: 8}}
```

Comment:
I know this was based on existing naming but it could be nice to rename `expectedEvent`s to `expectedConcludeEvent`, `expectedDepositEvent`,etc..?

Reply:
[575feff](https://github.com/statechannels/go-nitro/pull/1619/commits/575feffcd4d9994738653232cddda6c784654962)